sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs

```diff
@@ -47,6 +47,12 @@ public enum ServiceVersion
         /// </summary>
         internal ServiceVersion Version { get; }
 
+        /// <summary>
+        /// Gets or sets the Audience to use for authentication with Azure Active Directory(AAD).
+        /// The authentication scope will be set from this Audience.
+        /// </summary>
+        public TextAnalyticsAudience? Audience { get; set; }
```

Comment:
```suggestion
        public TextAnalyticsAudience Audience { get; set; } = TextAnalyticsAudience.AzureResourceManagerPublicCloud;
```

Reply:
My concern with doing this, rather than leaving <c>null</c> as the default is that it may be confusing to customers.   If I'm browsing the options and I see this URL value that I'm not familiar with, I may be confused and try to set the value to the URL of my Text Analytics resource.

On the other hand, if we leave this as nullable and use <c>null</c> as the default value, customers would be more likely to ignore it unless they know what it is needed for.

Reply:
Yeah, we should stick with a default of `null` so we have more wiggle room when we try to version in Johan's Cloud Config plan in the future.